distributions/test/pom.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
❓ Should we follow the DDF convention and define the project hierarchy for tests as: distributions/test/itests?

Reply:
I tried to address this one already. Is there something that I missed?

Reply:
Should there be another level underneath `test` named `itests` with its own pom file and the test class defined there?

Reply:
There is. The module is declared just below this comment 🙂

Reply:
Oops. I guess I just carried over this comment from the old repo 😆